src/sentry/api/helpers/group_index/update.py

```diff
@@ -840,13 +841,13 @@ def handle_is_public(
 
 
 def handle_assigned_to(
-    assigned_actor: str,
-    assigned_by: str,
-    integration: str,
+    assigned_actor: ActorTuple,
+    assigned_by: str | None,
+    integration: str | None,
```

Comment:
These types were wrong as we have tests passing in `None` and `ActorTuple` but never `str`.